src/shared/defaultAwsContextCommands.ts

```diff
@@ -97,7 +104,11 @@ export class DefaultAWSContextCommands {
             const profileName: string | undefined = await this.promptAndCreateNewCredentialsFile()
 
             if (profileName) {
-                await this._awsContext.setCredentialProfileName(profileName)
+                const successfulLogin = await UserCredentialsUtils.addUserDataToContext(profileName, this._awsContext)
+                if (!successfulLogin) {
+                    // credentials are invalid. Prompt user and log out
```

Comment:
// credentials are invalid. ~Prompt user and~ log out

Reply:
I'll probably drop a `// TODO: Prompt user` in, it's out of scope for this PR.